config/environments/development.rb

```diff
@@ -4,9 +4,7 @@
   config.after_initialize do
     Bullet.enable        = true
     Bullet.bullet_logger = true
-    Bullet.console       = true
     Bullet.rails_logger  = true
-    Bullet.add_footer    = true
```

Comment:
Couldn't find an obvious answer to have this inline script permitted by our csp
https://github.com/flyerhzm/bullet/issues/478

I think having rails logs is enough anyway unless someone strongly disagrees and wants to find a non hacky solution for this

Reply:
I think that's right — we'll always be able to see that in dev anyway.